### ISSUE TITLE
Develop infrastructure to support/retain BIPOC student 

### ISSUE THREAD

**Issue by sr320** [OARS-SAFS/DEI#43]
see 

- Why are there so few ethnic minorities in ecology and evolutionary biology? Challenges to inclusion and the role of sense of belonging 

- Does STEM Stand Out? Examining Racial/Ethnic Gaps in persistence Across postsecondary Fields

**Comment by sr320** [comment]
@JLGnotes did you say you had started compiling content for this?

**Comment by JLGnotes** [comment]
Yes, I have started compiling literature related to mentoring. I am thinking of starting with the recommendations/framework from [The Science of Effective Mentorship in STEMM](https://www.nap.edu/catalog/25568/the-science-of-effective-mentorship-in-stemm) (see below), and building off from there so that it'll be an approximately 5-page handout. I am sure there are many folks at SAFS who have lots of knowledge and experience related to mentoring and welcome help. 

Recommendations from The Science of Effective Mentorship in STEMM:

1) Adopt an Operational Definition of Mentorship in STEMM

2) Use an Evidenced-Based Approach to Support Mentorship

3) Establish and Use Structured Feedback Systems to Improve Mentorship at All Levels

4) Recognize and Respond to Identities in Mentorship

5) Support Multiple Mentorship Structures

6) Reward Effective Mentorship

7) Mitigate Negative Mentorship Experiences

8) Recommendations for Funding Agencies that Support Mentorship

9) Recommendations to Scholars of Mentorship

**Comment by JLGnotes** [comment]
Apologies, I jumped to the idea of the mentoring handout because it is more at the forefront of my mind, given that it was discussed at our last committee meeting. Also, 'was planning on another handout for onboarding and retention of underserved/minoritized groups. The mentoring handout could call out to the onboarding/retention handout. 

Below is a short list of references that may be useful for this topic.

Asai, D. J. (2020). Race Matters. Cell, 181(4), 754–757. https://doi.org/10.1016/j.cell.2020.03.044
Beltran, R. S., Marnocha, E., Race, A., Croll, D. A., Dayton, G. H., & Zavaleta, E. S. (n.d.). Field courses narrow demographic achievement gaps in ecology and evolutionary biology. Ecology and Evolution. https://doi.org/10.1002/ece3.6300

Blosser, E. (2020). An examination of Black women’s experiences in undergraduate engineering on a primarily white campus: Considering institutional strategies for change. Journal of Engineering Education, 109(1), 52–71. https://doi.org/10.1002/jee.20304

Dutt, K. (2020). Race and racism in the geosciences. Nature Geoscience, 13(1), 2–3. https://doi.org/10.1038/s41561-019-0519-z

Fovet, F. (2020). Beyond Novelty: “Innovative” Accessible Teaching as a Return to Fundamental Questions Around Social Justice and Reflective Pedagogy. https://www.igi-global.com/chapter/beyond-novelty/252132

Gautreaux, M. (2018). Towards shared understandings: An exploration of the literature on inclusive teaching and its implication for building Inclusive Excellence at UBC.

Kang, S. P., Chen, Y., Svihla, V., Gallup, A., Ferris, K., & Datye, A. K. (n.d.). Guiding change in higher education: an emergent, iterative application of Kotter’s change model. Studies in Higher Education, 20. https://doi.org/10.1080/03075079.2020.1741540

Kim, S. S., & Gelfand, M. J. (2003). The influence of ethnic identity on perceptions of organizational recruitment. Journal of Vocational Behavior, 63(3), 396–416. https://doi.org/10.1016/s0001-8791(02)00043-x

Knaub, A. V, Maier, S. J., & Ding, L. (2020). Changing Culture and Climate to Prevent Sexual Harassment in the Physics Educational Setting. Physics Teacher, 58(5), 352–355. https://doi.org/10.1119/1.5145535

O’Brien, L. T., Bart, H. L., & Garcia, D. M. (2020). Why are there so few ethnic minorities in ecology and evolutionary biology? Challenges to inclusion and the role of sense of belonging. Social Psychology of Education, 23(2), 449–477. https://doi.org/10.1007/s11218-019-09538-x

Quan, G. M., Corbo, J. C., Finkelstein, N. D., Pawlak, A., Falkenberg, K., Geanious, C., Ngai, C., Smith, C., Wise, S., Pilgrim, M. E., & Reinholz, D. L. (2019). Designing for institutional transformation: Six principles for department-level interventions. Physical Review Physics Education Research, 15(1), 22. https://doi.org/10.1103/PhysRevPhysEducRes.15.010141

Reinholz, D. L., Matz, R. L., Cole, R., & Apkarian, N. (2019). STEM Is Not a Monolith: A Preliminary Analysis of Variations in STEM Disciplinary Cultures and Implications for Change. Cbe-Life Sciences Education, 18(4). https://doi.org/10.1187/cbe.19-02-0038

Riegle-Crumb, C., King, B., & Irizarry, Y. (2019). Does STEM Stand Out? Examining Racial/Ethnic Gaps in Persistence Across Postsecondary Fields. Educational Researcher, 48(3), 133–144. https://doi.org/10.3102/0013189x19831006

Rosa, K., & Mensah, F. M. (2016). Educational pathways of Black women physicists: Stories of experiencing and overcoming obstacles in life. Physical Review Physics Education Research, 12(2). https://doi.org/10.1103/PhysRevPhysEducRes.12.020113

Smith, N. S., Côté, I. M., Martinez-Estevez, L., Hind-Ozan, E. J., Quiros, A. L., Johnson, N., Green, S. J., Cornick, L., Shiffman, D., Malpica-Cruz, L., Besch, A. G., & Shiel-Rolle, N. (2017). Diversity and inclusion in conservation: A proposal for a Marine Diversity Network. Frontiers in Marine Science, 4(AUG). https://doi.org/10.3389/fmars.2017.00234
